app/controllers/procurement_areas_controller.rb

```diff
@@ -50,4 +50,8 @@ def procurement_area_params
   def procurement_area
     @_procurement_area ||= ProcurementArea.find(params[:id])
   end
+
+  def organisations
+    all_organisations_of_type(types: {})
+  end
```

Comment:
When loading the organisations that belong_to a ProcurementArea it would be better to filter the organisations we need using the api rather than loading all organisations and then filtering in the ProcurementAreaPresenter.

Something like:
```
def organisations
    all_organisations_of_type(uids: @procurement_area.organisation_uids)
 end
```